ONSdigital/babbage
# Removed references to old homepage

### What

#### Removed references to the old homepage

- Java Endpoint
- T1 template file
- Partials used on the homepage
- Classes applied if page type is home_page
- Enum for page type
- Sparkline conditions for the homepage (highcharts)
- Control for which template to load
- Localisation
- Old data.json for the homepage

### How to review

- Go through the remaining code and ensure that nothing has been left that should have been removed.
- Ensure that any partials or components that have undergone changes still look and behave as they did before this (compendium, t4-3, pages that use standard-header, compendium landing page)
- Check that localisations that have been removed have no references to them (or they shouldn't have been removed)

### Who can review

Anyone except me
